packages/roosterjs-editor-core/test/coreApi/selectTableTest.ts

```diff
@@ -3,7 +3,7 @@ import { Browser } from 'roosterjs-editor-dom';
 import { EditorCore, TableSelection } from 'roosterjs-editor-types';
 import { selectTable } from '../../lib/coreApi/selectTable';
 
-describe('selectTable |', () => {
+xdescribe('selectTable |', () => {
```

Comment:
why?

Reply:
These tests seem to be unstable sometimes fail in the CI Test run. added an Issue here https://github.com/microsoft/roosterjs/issues/1237 to renable and try to stop using HTML string comparison, as that is what is causing the issue

Reply:
I can add the fix for those tests in this same PR too.